### PR TITLE
4709 schema org update

### PR DESCRIPTION
**Pull request by hyi**
### Pull Request Checklist: 
- [ ] Positive Test Case Written by Dev

- [ ] Automated Testing

- [ ] Sufficient User and Developer Documentation

- [ ] Passing Jenkins Build

- [ ] Peer Code review and approval

### Positive Test Case
1. [Enter positive test case here]

### CODE REVIEW

**Comment by hydrocheck** [comment]
| METRIC | VALUE |
|:-----:|:-----:|

[https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-4725](https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-4725)

File: hs_core/templates/pages/genericresource.html [hunk 1]
```diff
@@ -118,12 +116,17 @@
...
-    "funder": [
+    "funding": [
     {% for agency in fundingagencies %}
         {
-            "@type": "Organization",
-            "name": "{{ agency.agency_name | escapejs }}",
-            "url": "{{ agency.agency_url }}"
+            "@type": "Grant",
+            "identifier": "{{ agency.award_number }}",
+            "name": "{{ agency.award_title }}",
```
Comment (devincowan) [inline hs_core/templates/pages/genericresource.html:124]:
```suggestion
            "name": "{{ agency.award_title | escapejs }}",
```

**Review by devincowan — APPROVED** [review]
looks good! one suggestion for you to consider...

File: hs_core/templates/pages/genericresource.html (hunk 1 above)
Comment (hyi) [reply hs_core/templates/pages/genericresource.html]: @devincowan Good catch. Have just added the escapejs filter. See if it looks good now.

**Pull request merged**